javascript/javascript1/week3/song.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+const songDatabase = [{
+    songId: 1,
+    title: 'My baby',
+    artist: 'Soggy socks',
+  },
+  {
+    songId: 2,
+    title: '3 nails in wood',
+    artist: 'The carpenters',
+  },
+  {
+    songId: 3,
+    title: 'Blacker than black',
+    artist: 'Instant coffee',
+  },
+  {
+    songId: 4,
+    title: 'When is enough too little?',
+    artist: 'The spies girls',
+  }
+];
+
+
+function addSongToDatabase(song) {
+    const myPlaylist = [];
+    
+    songDatabase.push(song);
+    console.log(songDatabase);
+}
+
+function getSongByTitle(title) {
+  
+       const index = songDatabase.findIndex( function (songDatabase) {
```

Comment:
ok and what if index is not found (-1)? You have to check for that

Reply:
function getSongByTitle(title) {
  

  const index = songDatabase.findIndex( function (songDatabase) {
    return title === songDatabase.title ;
     })
     if ( index === -1)
    {
      return "Song not found";
    }
    else
    {
      return songDatabase[index];
    }
       
}

Reply:
yes

Reply:
Yes this works better. Also make your formatting a bit nicer, it is hard to read what is part of the function and what is not.